### PR TITLE
PADV-770: Add LICENSE, update README.rst

### PR DESCRIPTION
**Pull request by kuipumu**
## Tickets

- https://agile-jira.pearson.com/browse/PADV-770

## Description

This PR adds an LICENSE document to the repository and an update to the README.rst document.

## Type of Change

- Add LICENSE document.
- Update README.rst document.

## Reviewers

- [x] @alexjmpb
- [ ] @Jacatove

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -27,38 +26,111 @@ Installation on Open edX Devstack
...
   cd openedx-lti-tool-plugin
   virtualenv venv
   source venv/bin/activate
 
-- Install development dependencies.
+3. Install development dependencies.
 
 .. code-block:: bash
 
   make dev-requirements
```
Comment (Jacatove) [inline README.rst:57]: Could you please a step to run tests?. @kuipumu

File: README.rst [hunk 2]
```diff
@@ -27,38 +26,111 @@ Installation on Open edX Devstack
...
 
-Settings
-========
+LTI 1.3 Resource Launch Setup
+=============================
+
+1. (Optional) Expose the LMS service to an external domain.
```
Comment (Jacatove) [inline README.rst:62]: This step might not be clear for all people, could you please elaborate a little bit more?, don't know if you can name some of the tools you have used. @kuipumu

File: README.rst [hunk 3]
```diff
@@ -27,38 +26,111 @@ Installation on Open edX Devstack
...
+Optional Features
+*****************
+
+Course Access Configuration
+===========================
+
+By default, all resource links to all courses are allowed. If course access needs to be restricted you can set up a course access configuration policy for each LTI tool configuration deployed, this will allow you to set a list of allowed courses that an LTI tool configuration deployed is allowed for LTI 1.3 resource launches. Follow these next steps to set this feature:
+
+1. Enable the Django Waffle switch: `openedx_lti_tool_plugin.course_access_configuration`.
+2. Go to LMS Admin > Open edX LTI Tool Plugin > Course access configurations.
```
Comment (Jacatove) [inline README.rst:105]: @kuipumu I am getting a 500 when going to https://partner-courses.pearson.com/admin/openedx_lti_tool_plugin/courseaccessconfiguration/

File: README.rst [hunk 4]
```diff
@@ -27,38 +26,111 @@ Installation on Open edX Devstack
...
+3. On the configuration list, find the configuration that matches the previously created LTI tool.
+4. Edit the "Allowed Course IDs" field and add the courses that should be allowed.
+
+Complete Course Launch
+======================
+
+By default, the LTI tool doesn't support LTI 1.3 resource launches over a complete Open edX course, with this feature, you can enable resource launches over a whole Open edX course, this will redirect the launch user to the Open edX learning MFE for the requested course: Follow these next steps to set this feature:
+
+1. Enable the Django Waffle switch: `openedx_lti_tool_plugin.course_access_configuration`.
```
Comment (Jacatove) [inline README.rst:114]: I think this is not the waffle, isn't it?. @kuipumu

File: README.rst [hunk 5]
```diff
@@ -27,38 +26,111 @@ Installation on Open edX Devstack
...
+2. Setup an LTI 1.3 resource launch URL to a course (Example: http://localhost:18000/openedx_lti_tool_plugin/course/course-v1:x+x)
```
Comment (Jacatove) [inline README.rst:115]:
```suggestion
2. Setup an LTI 1.3 resource launch URL to a course (Example: http://localhost:18000/openedx_lti_tool_plugin/course/course-v1:ORG+RUN)
```

**Review by Jacatove — CHANGES_REQUESTED** [review]

File: README.rst (hunk 3 above)
Comment (kuipumu) [reply README.rst:105]: @Jacatove I found the error in the logs, the LMS is missing migrations

File: README.rst (hunk 4 above)
Comment (kuipumu) [reply README.rst:114]: @Jacatove This is the switch, I just verified on the code

**Review by Jacatove — APPROVED** [review]

**Pull request merged**